tests/Unit/Core/SuggestionTest.php

```diff
@@ -48,7 +48,8 @@ public function testCastsToArray()
             'class_import' => 'Namespace\\Foobar',
             'documentation' => 'foo',
             'label' => 'hallo',
-            'range' => Range::fromStartAndEnd(1, 2)
+            'range' => Range::fromStartAndEnd(1, 2),
+            'snippet' => 'FooBar(${1:\$label})$0', // For test purposes
```

Comment:
Extract to a constant (`EXAMPLE_SNIPPET`) ?

Reply:
Why ?
I could even remove this, it's not implementing yet so I have no reason to test it :)
Kind of strange that it works even, I will have to check it.

Reply:
just because you are asserting it below, and it's a non-trivial string.